frontend/public/components/utils/request-size-input.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as React from 'react';
+import { Dropdown } from '.';
+
+export class RequestSizeInput extends React.Component<RequestSizeInputProps> {
+  onChange = event => {
+    // Need this logic to handle the different input from dropdown vs input
+    let obj = {};
+    if (typeof event !== 'object') {
```

Comment:
It would be better to just have separate handlers instead of trying to inspect the type of the value.